sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -269,7 +269,9 @@
 
   /**
    * @since 2.11
+   * @deprecated since 6.7
    */
+  @Deprecated
   String CPD_CROSS_PROJECT = "sonar.cpd.cross_project";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)